airflow/contrib/operators/ssh_operator.py

```diff
@@ -99,6 +99,8 @@ def execute(self, context):
                 if self.command.startswith('sudo'):
                     get_pty = True
 
+                self.logger.info("Running command: {0}".format(self.command))
```

Comment:
`self.logger` is deprecated in favour of `self.log`.

Also, with `"".format()` Python constructs the string beforehand so with logging this form is preferred: `self.log.info("Running command: %s", self.command)` which constructs a string only when this statement is evaluated.